src/lib/mergeSchemas.js

```diff
@@ -45,10 +46,17 @@ export function createConvectionLink() {
 }
 
 export async function mergeSchemas() {
+  // The below all relate to Convection stitching.
+  // TODO: Refactor when adding another service.
+  const convectionTypeDefs = fs.readFileSync(
+    path.join("src/data/convection.graphql"),
+    "utf8"
+  )
+
   const convectionLink = createConvectionLink()
 
   const convectionSchema = await makeRemoteExecutableSchema({
-    schema: await introspectSchema(convectionLink),
+    schema: convectionTypeDefs,
```

Comment:
This is the essential replacement in this PR, instead of introspecting the schema we use the types generated in the file.